src/ServiceControl.Config/Commands/AddInstanceCommand.cs

```diff
@@ -5,21 +5,31 @@
 
 namespace ServiceControl.Config.Commands
 {
+    using ServiceControl.Config.Framework.Modules;
+
     class AddInstanceCommand : AbstractCommand<object>
     {
         private readonly Func<InstanceAddViewModel> addInstance;
         private readonly IWindowManagerEx windowManager;
+        private readonly Installer installer;
 
-        public AddInstanceCommand(IWindowManagerEx windowManager, Func<InstanceAddViewModel> addInstance) : base(null)
+        public AddInstanceCommand(IWindowManagerEx windowManager, Func<InstanceAddViewModel> addInstance, Installer installer) : base(null)
         {
             this.windowManager = windowManager;
             this.addInstance = addInstance;
+            this.installer = installer;
         }
 
         public override void Execute(object obj)
         {
+            var licenseCheckResult = installer.CheckLicenseIsValid();
+            if (!licenseCheckResult.Valid)
+            {
+                windowManager.ShowMessage(licenseCheckResult.Message, "Installation could not continue due to an issue with the current license. Contact sales@particular.net", hideCancel: true);
```

Comment:
@gbiellem, it seems the error message `licenseCheckResult.Message` is actually displayed as the title of the popup dialog, shouldn't the `licenseCheckResult.Message` be part of the body ?